tests/console/zypper_extend.pm

```diff
@@ -65,6 +79,42 @@ sub run {
     #Remove star, don't ask for permission:
     zypper_call 'rm star';
 
+    #Install specific versioni of a package
+    my $version = script_output q[zypper se -s star |grep " star " |grep x86_64 | awk 'END {print $6}'];
+    zypper_call 'in -f star-$version';
+
+    #Install and remove package combination
+    zypper_call 'in vim -star';
+    assert_script_run("rpm -q vim | grep vim- ");
+    assert_script_run("rpm -q star | grep not");
+
+    #Cleaning up dependencies of removed packages
+    zypper_call 'rm --clean-deps vim';
+    assert_script_run("rpm -q vim | grep not");
+
+    #Add a repository
+    zypper_call "ar -p 90 -f --no-gpgcheck http://ftp.gwdg.de/pub/linux/misc/packman/suse/openSUSE_Tumbleweed/ packman";
```

Comment:
Just small note for being consistent :) 
Mostly you use `''` but then without reason like use of variable you use `""`

Reply:
> Just small note for being consistent :)
> Mostly you use `''` but then without reason like use of variable you use `""`

You're right, I will modify it